src/applications/disability-benefits/all-claims/tests/utils/dates/validations.unit.spec.jsx

```diff
@@ -221,4 +222,276 @@ describe('Disability benefits 526EZ -- Date validation utilities', () => {
       expect(errors.addError.called).to.be.false;
     });
   });
+
+  describe('validateApproximateDate', () => {
+    describe('with allowPartial=true (default)', () => {
+      it('should accept year-only format', () => {
+        const errors = { addError: sinon.spy() };
+        validateApproximateDate(errors, '2020-XX-XX');
+        expect(errors.addError.called).to.be.false;
+      });
+
+      it('should accept year-month format', () => {
+        const errors = { addError: sinon.spy() };
+        validateApproximateDate(errors, '2020-06-XX');
+        expect(errors.addError.called).to.be.false;
+      });
+
+      it('should accept full date format', () => {
+        const errors = { addError: sinon.spy() };
+        validateApproximateDate(errors, '2020-06-15');
+        expect(errors.addError.called).to.be.false;
+      });
+
+      it('should reject month-only format (no year)', () => {
+        const errors = { addError: sinon.spy() };
+        validateApproximateDate(errors, 'XXXX-06-XX');
+        expect(errors.addError.called).to.be.true;
+        expect(errors.addError.firstCall.args[0]).to.include(
+          'Enter a year only',
```

Comment:
I would interpret this error to imply I can only enter a year value, even though there is a year, month and day field provided. Is that correct? Same question applies for the content of the others errors in this block.